src/components/PaymentLog.vue

```diff
@@ -1271,12 +1323,16 @@ export default {
     },
     getCalc() {
       this.watchCashPrice()
-      
-      if(this.salesLogForm.business_type_id?.slug.includes('product') || this.salesLogForm.business_type_id?.slug === 'ap_cash_n_carry' || this.salesLogForm.business_type_id?.slug.includes('products')){ 
-          this.serial = true
-        }else{
-          this.serial = false
-        }
+
+      if (
```

Comment:
We can simplify to below

`this.serial = ["product", "ap_cash_n_carry", "products"].includes(this.salesLogForm.business_type_id?.slug)`